platform/linux-generic/include/odp_bitset.h

```diff
@@ -65,12 +53,6 @@ static inline uint32_t bitset_ffs(bitset_t b)
 		return 0;
 }
 
-/* Load-exclusive with memory ordering */
-static inline bitset_t bitset_monitor(bitset_t *bs, int mo)
-{
-	return monitor128(bs, mo);
```

Comment:
It appears that this commit "linux-gen: wait_until: add generic bitset wait until function" also fixes a bug in the arm architecture implementation: monitor128() is erroneously implemented using lld(), which operates on 64 bit data (unlike the lld() in aarch64).

So, maybe the commit message could explain that this is also a bug fix. Or maybe the fix could be in a separate commit (but maybe it is not very practical because the fix basically requires disabling of monitor128 and thus the wfe loop in arm).